lib/tests/test_local_working_copy.rs

```diff
@@ -565,7 +565,13 @@ fn test_tree_builder_file_directory_transition() {
 
     // Add file at parent_path
     let mut tree_builder = store.tree_builder(store.empty_tree_id().clone());
-    testutils::write_normal_file(&mut tree_builder, parent_path, "");
+    tree_builder.set(
+        parent_path.to_owned(),
+        TreeValue::File {
+            id: testutils::write_file(store, parent_path, ""),
+            executable: false,
+        },
+    );
```

Comment:
nit: maybe we shouldn't remove `write_normal_file()`, etc.? They seems to be useful.

Reply:
These three uses in `test_local_working_copy.rs` were in fact the only ones. I would argue that the goal here is to test the `TreeBuilder` itself, so it is better to "inline" this helper so the calls to the tested `TreeBuilder` APIs become apparent.

Additionally, the helpers `write_normal_file()`, etc., operated on another different level compared to `write_file()`. The latter really only writes the object to the store, whereas `write_normal_file()` does that too _and_ sets the corresponding `TreeValue` on the `TreeBuilder`. That's not really apparent from the naming.